grumphp.yml.dist

```
@@ -13,9 +13,9 @@ parameters:
 
   extra_tasks:
     phpparser:
-      ignore_patterns: %tasks.phpcs.ignore_patterns%
+      ignore_patterns: [ tasks.phpcs.ignore_patterns% ]
```

Is it should be like that because in other components adapted to D9 I see other syntaxis?
```suggestion
      ignore_patterns: "%tasks.phpcs.ignore_patterns%"
```

---

not yet addressed as I can see.
Please see oe_media D9 version how it is done: `ignore_patterns: "%tasks.phpcs.ignore_patterns%"`